components/theme/footer.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Image from 'next/image'
+
+import styles from './footer.module.scss'
+
+export default function Footer() {
+  return (
+    <div className={styles.footer}>
+      {/* Title */}
+      <h5 className={styles.title}> 
+        Vườn hoa sứ
+      </h5>
+      
+      {/* Icon */}
+      <div className={styles.iconContainer}>
+        <div className={styles.icon}><Image src='/icons/facebook.svg' alt='facebook' layout='fill' objectFit='contain'/></div>
```

Comment:
Please use the library "[react-icons](https://react-icons.github.io/react-icons/)" for this